common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -406,7 +406,7 @@ div.problem {
       margin-top: 3px;
 
       .MathJax_Display {
-        display: inline-block;
+        display: inline-block !important;
```

Comment:
Do you know where else this rule is used? Would it be possible to scope this just to the formula equation input type?